bats/tests/helpers/vm.bash

```diff
@@ -45,7 +45,7 @@ macos_eject_ramdisk() {
     run hdiutil info -plist
     assert_success
     # shellcheck disable=2154 # $output set by `run`
-    run plutil -convert json -o - - <<<"$output"
+    run --separate-stderr plutil -convert json -o - - <<<"$output"
```

Comment:
Again, we should either use `run_without_error`, or document which potential warnings/errors we are intentionally ignoring.